zineb/models/constraints.py

```diff
@@ -23,21 +23,49 @@ def __call__(self, value_to_check):
         if len(self.constrained_fields) == 1:
             field = self.constrained_fields[-1]
             counter.update(self._data_container.columns.as_values()[field])
-            
+
             element_count = counter.get(value_to_check, 0)
             if element_count > 1:
                 errors.extend(
                     [(field, ConstraintError(self.model._meta.model_name, self.name))]
                 )
         else:
-            for field in self.constrained_fields:
-                counter.update(self._data_container.columns.as_values()[field])
-
-            element_count = counter.get(value_to_check, 0)
-            if element_count == 1:
-                errors.extend(
-                    [(self.constrained_fields, ConstraintError(self.model._meta.model_name, self.name))]
-                )
+            # TODO: Think of a workflow that can validate
```

Comment:
We should remove the whole idea of unique together from the UniqueConstraint flow due to the complexity of checking at the same time if two concurrent values match in the database